cellprofiler/utilities/version.py

```diff
@@ -92,10 +92,23 @@ def get_version():
         import cellprofiler.frozen_version
         return cellprofiler.frozen_version.version_string
 
+def get_dotted_version():
```

Comment:
Never have seen a SHA-1 published. Mark knows the SHA-1 of every CellProfiler release though - if someone posts a pipeline on the forum, he figures out which version of CellProfiler from that.